cinema_service/settings.py

```diff
@@ -134,7 +139,7 @@
 STATIC_URL = "static/"
 
 MEDIA_URL = "/media/"
-MEDIA_ROOT = BASE_DIR / "media"
+MEDIA_ROOT = "/files/media"
```

Comment:
The MEDIA_ROOT setting is using an absolute path, which might cause issues if the directory /files/media does not exist on the server. Consider using a path relative to the BASE_DIR.